.github/workflows/test-e2e.yml

```diff
@@ -66,11 +62,11 @@ jobs:
       - name: Test Application End to End
         id: 'test_e2e'
         env:
-          BROWSERS: firefox
+          BROWSERS: ${{matrix.browser}}
         run: npm run test:e2e
       - name: Upload e2e test results
         if: ${{ failure() && steps.test_e2e.outcome == 'failure' }}
         uses: actions/upload-artifact@v2
         with:
-          name: e2e-firefox-test-results-${{matrix.os.name}}-${{github.sha}}
+          name: e2e-${{matrix.browser}}-test-results-${{matrix.os}}-${{github.sha}}
```

Comment:
As `matrix.browser` is an array, we should not specify it in the artifact name ?
```suggestion
          name: e2e-test-results-${{matrix.os}}-${{github.sha}}
```

Reply:
This is the correct syntax as suggested by the GH Actions documentation: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix. `browser` is an element of the matrix and will be a string in the matrix context available in the job.

I have introduced a fake error in fd59cd4 and the artifact containing the e2e test reports have a correct name
![image](https://user-images.githubusercontent.com/27200110/112620188-76ecd280-8e28-11eb-8912-9faf75f6178b.png)